src/handler/http/request/folders.rs

```diff
@@ -74,17 +104,21 @@ pub async fn create_folder(
         }),
     ),
     responses(
-        (status = StatusCode::CREATED, description = "Folder updated", body = Folder),
+        (status = StatusCode::OK, description = "Folder updated", body = HttpResponse),
```

Comment:
The old description of the success response was incorrect. As mentioned [here](https://github.com/openobserve/openobserve/pull/5206/files#r1865942378) previously this route returned a 200 (OK) response with a success message, but no folder object in its response body.

This PR the old behavior of the endpoint. I'm just updating the docs to reflect it's actual behavior.